### PR TITLE
New Blog Post: Senior executives part 2

### PR DESCRIPTION
**Pull request by colinmurphy01**
# Pull request summary
Creates a new post for the blog entitled Senior executives part 2: work outside the traditional hierarchy to set up an initiative for success

## Reminder - please do the following before assigning reviewer
- [ ] For frontend changes, ensure design review (@jstrothman)

And make sure that automated checks are ok

- fix houndci feedback
- ensure tests pass
- federalist builds
- no new SNYK vulnerabilities are introdcued

### CODE REVIEW

File: _posts/2022-08-02-senior-executives-pt2.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  - allison-press
+  - jeffrey-durland
+tags:
+  - senior executives
+  - stakeholder management
+  - modern practices
+  - agile
+excerpt: "In order to ultimately succeed at delivering human-centered software, two critical groups need to work closely together: the product team and a champions group. This is part two in a series on how senior executives and tech teams can be better allies."
+---
+_This series examines how teams need to work closer with senior executives as allies. ([see part 1]({{ site.baseurl }}/2022-07-20-senior-executives-pt1.md))_
```
Comment (Sgtpluck) [inline _posts/2022-08-02-senior-executives-pt2.md:15]: this should be `/07/20/senior-executives-pt1/` i think

File: _posts/2022-08-02-senior-executives-pt2.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+**The goal is to make strategic decisions based on user feedback.** This requires the tightest possible feedback loop between a cross-functional product team leading the work and a small group of champions overseeing its success. Otherwise, you'll spend more time justifying the work rather than doing it.
+
+A cross-functional product team: 
+-   Consists of five to nine dedicated people, with the user research, design, content, software development, and/or security skills necessary to get the job done
+-   Has a full-time product owner leading the team, who can make decisions and is accountable to the team, organization, and end users
+-   Focuses initially on understanding user needs, validating the path forward, assessing risks, building the new product, and validating and testing the product strategy
+-   Costs between $1-4M in the first year (depending partly on whether they need clearances). A more detailed cost estimate is in [18F's guide to de-risking government technology projects]([url](https://derisking-guide.18f.gov/federal-field-guide/planning/#software-development-efforts-should-be-tightly-scoped-to-reduce-risk-and-avoid-overspending))
```
Comment (Sgtpluck) [inline _posts/2022-08-02-senior-executives-pt2.md:27]: if you remove this [url] piece i think this should work?

```
[18F's guide to de-risking government technology projects](https://derisking-guide.18f.gov/federal-field-guide/planning/#software-development-efforts-should-be-tightly-scoped-to-reduce-risk-and-avoid-overspending)
```

File: _posts/2022-08-02-senior-executives-pt2.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  - allison-press
+  - jeffrey-durland
+tags:
+  - senior executives
+  - stakeholder management
+  - modern practices
+  - agile
+excerpt: "In order to ultimately succeed at delivering human-centered software, two critical groups need to work closely together: the product team and a champions group. This is part two in a series on how senior executives and tech teams can be better allies."
+---
+_This series examines how teams need to work closer with senior executives as allies. ([see part 1]({{site.baseurl}}/07/20/senior-executives-pt1/))_
```
Comment (Sgtpluck) [inline _posts/2022-08-02-senior-executives-pt2.md:15]: Sorry, I left off the year.

`2022/07/20/senior-executives-pt1/`

**Review by jstrothman — COMMENTED** [review]
Ready to merge! 
Holding off merging until @colinmurphy01 comments on leaving table as is

**Comment by colinmurphy01** [comment]
@jstrothman looks good to me

**Pull request merged**